includes/Main.php

```diff
@@ -1770,6 +1773,80 @@ function_exists( 'WPO_WCPDF_Templates' ) &&
 			</tr>';
 		}
 	}
+
+	/**
+	 * Get WC emails as array of id => title.
+	 *
+	 * @return array
+	 */
+	public function get_wc_emails(): array {
```

Comment:
There are other instances where we retrieve WC emails, so it might be better to move this to the functions file.